Source/Meadow.Foundation.Peripherals/Sensors.Temperature.Thermistor/Samples/Thermistor_Sample/MeadowApp.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Meadow;
+using Meadow.Devices;
+using Meadow.Foundation.Sensors.Temperature;
+using Meadow.Units;
+using System;
+using System.Threading.Tasks;
+
+namespace Sensors.Temperature.Thermistor
```

Comment:
namespace Sensors.Temperature.Thermistor_Sample (important for Mirid to process this file correctly)